torch/export/__init__.py

```diff
@@ -349,6 +349,7 @@ def export(
     *,
     constraints: Optional[List[Constraint]] = None,
     dynamic_shapes: Optional[Union[Dict[str, Any], Tuple[Any]]] = None,
+    strict: bool = True,
```

Comment:
Since "strict" is user-facing. We might need to add warnings and docs for this "strict" keyword. It would be better if we can mention what are the observable consequences/implications of using strict.

Reply:
Maybe we should make it "_strict"?

Reply:
I don't think this matters because non strict mode is already unsafe by design, and we just need to fix forward bugs if there's any.